common/test/unit/com/thoughtworks/go/domain/materials/git/GitTestRepo.java

```diff
@@ -67,7 +76,7 @@ public GitTestRepo(File gitBundleFile) {
     }
 
     public String projectRepositoryUrl() {
-        return gitRepo.getAbsolutePath();
+        return FileUtil.toFileURI(gitRepo);
```

Comment:
This change make projectRepositoryUrl method return a real url (`file:///foo/bar.git` instead of `/foo/bar.git`). The reason I switch it is because git clone depth option can not be used with a local file system clone.